### DISCUSSION TITLE
Meshcentral 2 for linux and npm

### DISCUSSION THREAD

**Discussion by EdwenSkylar** [Ylianst/MeshCentral#6101] [General]
hello, I don't see the meshcentral2 version available in final or beta version with npm?  I'm on Linux.

**Comment by si458** [comment]
The is no meshcentral2.
Meshcentral was v1, then rewritten to become v2, 
But then the naming changed back to meshcentral v1.x
The current version is meshcentral 1.1.22

**Reply by EdwenSkylar** [reply]
Ok Thanks.
